dotCMS/src/main/java/com/dotmarketing/portlets/categories/business/CategoryFactoryImpl.java

```diff
@@ -736,6 +736,8 @@ public void sortTopLevelCategories()  throws DotDataException {
 	}
 
 	public void sortChildren(String inode)  throws DotDataException {
+        inode = SQLUtil.sanitizeParameter( inode );

```

Comment:
@oarrietadotcms I just walked through the changes needed and, even though they are a bit tedious, I vote for changing to PreparedStatements